charts/jaeger/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart for the jaeger add-on in Linkerd
+name: jaeger
+version: 0.1.0
+maintainers:
+  - name: Linkerd authors
+    email: cncf-linkerd-dev@lists.cncf.io
+    url: https://linkerd.io/
```

Comment:
unless the Helm convention is otherwise, we should always end files with a newline